http_logs/operations/default.json

```diff
@@ -62,15 +62,31 @@
       "name": "term",
       "operation-type": "search",
       "index": "logs-*",
+      "request-timeout": 7200,
       "body": {
         "query": {
           "term": {
-            "request.raw": {
-              "value": "GET / HTTP/1.0"
-            }
+            {%- if runtime_fields is defined %}
+              {# Runtime fields make a "path.raw" instead of "request.raw" because that is what the canned example of grok and dissect typically do. #}
+              "path.raw": {
+                "value": "/"
+              }
+            {%- else %}
+              "request.raw": {
+                "value": "GET / HTTP/1.0"
+              }
+            {%- endif %}
           }
         }
-      }
+      },
+      "detailed-results": true,
+      "assertions": [
+        {
+          "property": "hits",
+          "condition": "==",
+          "value": 10000
```

Comment:
I've tested this and it seems to be working fine in all cases, but just to double check this is the intention: this assertion will be imposed on all invocations of this term query (regardless of `runtime_fields`) i.e. also challenges like `append-no-conflicts` (also part of the nightlies), so it's intentional that the assertion on hits applies to all.

Similar comment for the `range` query below.

Reply:
Correct. I'd love to have a more precise assertion but this'll catch those cases where the we totally fail. Or somehow partially fail in a way that finds very few hits.